rxjs-1/util.js

```diff
@@ -0,0 +1,6 @@
+
+function round(num) {
```

Comment:
You were probably searching for [toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) method here 🙂

Reply:
Yes `.toFixed(2)` simply do the job. I don't remember why I used this custom method...